### ISSUE TITLE
Not identifying a textarea with no label as an issue

### ISSUE THREAD

**Issue by delphinegilbert** [microsoft/accessibility-insights-web#7117]
Is it normal that Accessibility Insights doesn't identify a textarea without a label as an issue? The ChatGPT (https://chat.openai.com/) website has a textarea with no labels and it doesn't show as a failure in the automated check or in the native widget section. See screenshots below. Thank you

![image](https://github.com/microsoft/accessibility-insights-web/assets/59966346/da70daaf-ca99-43f4-aaf3-dd362ec5ca23)
![image](https://github.com/microsoft/accessibility-insights-web/assets/59966346/f47c46d2-5eae-4761-a46f-8006ed7a8fbf)
![image](https://github.com/microsoft/accessibility-insights-web/assets/59966346/9a10b10f-fc67-47df-93af-e648b9ac460f)
![image](https://github.com/microsoft/accessibility-insights-web/assets/59966346/899d7b4d-2957-4ab0-b836-d69e16f4ec3f)

**Comment by v-singhanjal** [comment]
We acknowledged the issue and we are looking into it.

**Comment by v-viyada** [comment]
@delphinegilbert 
The text area contains a non empty placeholder that is why accessibility test for label is passing. Please refer https://accessibilityinsights.io/info-examples/web/label/ for more information on label.
